### ISSUE TITLE
Deprecation of mcnp5 patches 1.40 and 1.51

### ISSUE THREAD

**Issue by makeclean** [svalinn/DAGMC#370]
@ljacobson64 needs to make a new MCNP5 v1.6 patch to fix a bug in continue runs, this begged the question should he also regenerate patches for v1.4 and v1.51. If no, then we should deprecate the patches due to them being out of sync with the v1.6 patch. If yes, then obviously he should just go ahead and do it.

**Comment by ljacobson64** [comment]
I don't even see the original v1.4 or v1.51 source code in MCNP.git which we would need to diff against. There is only one branch that contains original source code and it's called "vendor-5.1.60-updated-dir".

Furthermore, I generated a new patch for v1.6 and found that it was significantly different than the one currently in svalinn/DAGMC. Assuming that MCNP.git contains up-to-date source code, the only difference between the current patch and my new patch should be the one line I changed in fmesh_mod.F90.
